pam/internal/adapter/authentication.go

```diff
@@ -397,6 +397,7 @@ func (m *authenticationModel) Compose(brokerID, sessionID string, encryptionKey
 	m.errorMsg = ""
 
 	if m.clientType != InteractiveTerminal {
+		m.currentModel = &focusTrackerModel{}
```

Comment:
I'm skeptical of using a `tea.Model` for tracking the state when we're not even using bubbletea for the user interface. Could we instead add an interface which can be implemented either by using `tea.Model` (or an abstraction which uses it) when we're using bubbletea or an internal model if we're not not using bubbletea?

Reply:
Works for me.